### ISSUE TITLE
How to load StrongSORT ReID model?

### ISSUE THREAD

**Issue by seermer** [mikel-brostrom/yolo_tracking#1154]
### Search before asking

- [X] I have searched the Yolo Tracking [issues](https://github.com/mikel-brostrom/yolo_tracking/issues) and found no similar bug report.

### Question

I am trying to load the ReID model used by the StrongSORT paper (BoT-ResNet50), the model file is provided by link from the official StrongSORT repo https://drive.google.com/drive/folders/1Zk6TaSJPbpnqbz1w4kfhkKFCEzQbjfp_ called DukeMTMC_BoT-S50.pth.

I have renamed it to DukeMTMC_BoT-resnet50.pt after download so it can be recognized as resnet50, but I received

> WARNING  | boxmot.appearance.reid_model_factory:load_pretrained_weights:201 - The pretrained weights "ckpts/DukeMTMC_BoT-resnet50.pt" cannot be loaded, please check the key names manually (** ignored and continue **)

Is the official ReID model not supported? How can I load it manually?

**Comment by mikel-brostrom** [comment]
As stated in the [paper](https://arxiv.org/pdf/1906.08332.pdf): 

> "We propose a novel neck
> structure named as batch normalization neck (BNNeck). BNNeck
> adds a batch normalization layer after global pooling layer
> to separate metric and classification losses into two different
> feature spaces because we observe they are inconsistent in one
> embedding space."

So it is not a pure resnet50. This model would have to be added.

**Comment by seermer** [comment]
Thank you so much for the clarification!
